example/linux/Makefile

```diff
@@ -140,4 +125,4 @@ $(FLUTTER_ASSETS_SOURCE):
 clean:
 	rm -rf $(OUT_DIR); \
 	cd $(FLUTTER_APP_DIR); \
-	$(FLUTTER_BIN) clean
+	$(FLUTTER_BIN) clean
```

Comment:
Add missing newline please (to avoid churn next time I edit it, since my editors are generally configured to add them)